scout/server/blueprints/institutes/templates/overview/cases.html

```diff
@@ -29,7 +29,7 @@
       </div>
       <div class="col-4">
           <label class="control-label" for="search_term">Search cases</label>
-          <input class="form-control" id="search_term" name="search_term" placeholder="example:18201" type="text" pattern="[^<>?!=\\/]*" title="Characters ^<>?!=\/ are not allowed">
+          <input class="form-control" id="search_term" name="search_term" value="{{form.search_term.data}}" placeholder="example:18201" type="text" pattern="[^<>?!=\\/]*" title="Characters ^<>?!=\/ are not allowed">
```

Comment:
This is to display what was searched after the form is submitted and the results are returned

Reply:
✅